CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 #### Promtail
 
 ##### Enhancements
+* [8953](https://github.com/grafana/loki/pull/8953) **dannykopping**: Querier: block queries by hash.
```

Comment:
That should go into the `Loki` section, right?

Reply:
Yes! Well spotted, thanks